### PR TITLE
Added counting sort in ruby

### LINKED ISSUE #694: Counting_Sort algo in ruby
it would be good to have counting sort algorithm in ruby language too.
sir i would like you` to assign this issue to me.

### PR DESCRIPTION
**Pull request by AnshitaVishwa**
Fixes #694 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding convetions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
implemented the counting sort algorithm using a different language.
- Added counting sort in ruby

#### Languages Used:
- Ruby

#### Files Added:
- countingsort.rb

### CODE REVIEW

**Comment by jainaman224** [comment]
@AnshitaVishwa Please fill the PR form first.

**Comment by AnshitaVishwa** [comment]
sir i have been assigned issue number #694 and in that i have implemented the counting sort algorithm in ruby language .

**Comment by jainaman224** [comment]
Basically you have to fill the PR template first.

**Comment by AnshitaVishwa** [comment]
sir i have filled the PR form

File: Counting_Sort/countingsort.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+
```
Comment (jainaman224) [inline Counting_Sort/countingsort.rb]: Fix file name to `Counting_Sort.rb`.

File: Counting_Sort/countingsort.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+def counting_sort(input , n)
+  max = input.max
```
Comment (jainaman224) [inline Counting_Sort/countingsort.rb]: Please add some comments to explain the algorithm.

File: Counting_Sort/countingsort.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  min = input.min
+  arr = []
+  for i in min..max
+    arr.push(0)
+  end
+  for i in input
+    arr[i-min]=arr[i-min]+1
```
Comment (jainaman224) [inline Counting_Sort/countingsort.rb]: Please add space around the `=`.

**Comment by AnshitaVishwa** [comment]
sir i have done the required changes in the file .

**Comment by AnshitaVishwa** [comment]
sir i have done the changes .

**Pull request merged**